src/servers/Cargo.toml

```
@@ -9,7 +9,7 @@ aide = { version = "0.9", features = ["axum"] }
 arrow-flight.workspace = true
 api = { path = "../api" }
 async-trait = "0.1"
-axum = "0.6"
+axum = { version = "0.6", features = ["form"] }
```

"form" feature is default?
```suggestion
axum = "0.6"
```